components/list/test/list.visual-diff.html

```diff
@@ -238,7 +238,7 @@
 			<div class="visual-diff" id="selectable">
 				<d2l-list>
 					<d2l-list-item label="Item 1" selectable key="1">Item 1</d2l-list-item>
-					<d2l-list-item label="Item 2" disabled selectable key="2">Item 2</d2l-list-item>
+					<d2l-list-item label="Item 2" selection-disabled selectable key="2">Item 2</d2l-list-item>
```

Comment:
It would also be a good idea to add unit tests for `selection-disabled` in [list-item-checkbox-mixin.test.js](https://github.com/BrightspaceUI/core/blob/main/components/list/test/list-item-checkbox-mixin.test.js)

Reply:
Should I leave the checks for disabled even though it is a deprecated property?

Reply:
I think that would be a good idea, so we can ensure that the property is still working as intended. Those tests can be removed when the property is being completely removed after the existing uses are switched over.

Reply:
Ok, I added a test to check that `selection-disabled` is added even when only `disabled` is set. I think is sufficient and cleaner, hence easier to remove when the property is removed.